addons/60fps/userscript.js

```diff
@@ -21,7 +21,7 @@ export default async function ({ addon, global, console }) {
         e.cancelBubble = true;
         e.preventDefault();
         mode = !mode;
-        if (mode) setFPS(addon.settings.get("framerate"));
+        if (mode) setFPS(Math.min(240, Math.max(31, addon.settings.get("framerate") | 31)));
```

Comment:
Surely that's not supposed to be a bitwise or?

Reply:
whoops